app/models/message.rb

```diff
@@ -23,6 +23,13 @@ class Message < ActiveRecord::Base
 
   before_validation :format_phone_number
 
+  enum honorific: {
+    'Ms.' => 0,
```

Comment:
Not that you have to, but you can do something like:

"Ms." => "Ms."

That way the db value is readable. The trade off is that integers are probably faster and smaller to store. In terms of indexing the db based on 4 strings or 4 ints it's pretty negligible.